exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,18 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) 
+      return 'A'
+  else
+  if (mark > 60) 
+      return 'B'
+  else
+  if (mark >= 50) 
+      return 'C'
+  else
+      return 'F';
+ }
```

Comment:
I actually didn't have a clue that you can skip internal curly brackets within if/else if statements and the function would still work. Thanks for enlightening me:)